### PR TITLE
Trev8939/scriptfixes

### PR DESCRIPTION
**Pull request by TADraeseke**
This PR handles various fixes made to the samples in order to make the sample viewer work. They are broadly:

- Adding categories to READMEs (where they are is unlikely to be permanent home, but makes the script work for now)
- Fix conflicting generic resource names on samples (for instance "data_sdcard_offline_dir" to "raster_folder" in the case that its pointing to samples/rasters)
- Fix to take screenshot which was broken post android sdk 24+)

### CODE REVIEW

File: java/feature-layer-update-attributes/README.md [hunk 1]
```diff
@@ -1,4 +1,5 @@
-# Feature layer update attributes         
+# Feature layer update attributes
+### Category: Manage Data
 This sample demonstrates how to edit attributes of feature in a feature layer.      
 
 ![Feature Layer Update Attributes App](feature-layer-update-attributes.png)
```
Comment (shellygill) [inline java/feature-layer-update-attributes/README.md]: I know it's not caused by your changes, but the 'How it works' subtitle is incorrectly formatted for display.

File: java/generate-geodatabase/src/main/java/com/esri/arcgisruntime/sample/generategeodatabase/MainActivity.java [hunk 2]
```diff
@@ -54,6 +50,9 @@
 import com.esri.arcgisruntime.tasks.geodatabase.GenerateGeodatabaseParameters;
 import com.esri.arcgisruntime.tasks.geodatabase.GeodatabaseSyncTask;
 
+import java.io.File;
+import java.util.concurrent.ExecutionException;
```
Comment (shellygill) [inline java/generate-geodatabase/src/main/java/com/esri/arcgisruntime/sample/generategeodatabase/MainActivity.java]: This code change is not consistent with the code guidelines.

File: java/open-mobile-mappackage/src/main/java/com/esri/arcgisruntime/sample/openmobilemappackage/MainActivity.java [hunk 3]
```diff
@@ -28,11 +26,12 @@
 import android.support.v7.app.AppCompatActivity;
 import android.util.Log;
 import android.widget.Toast;
-
 import com.esri.arcgisruntime.loadable.LoadStatus;
 import com.esri.arcgisruntime.mapping.MobileMapPackage;
 import com.esri.arcgisruntime.mapping.view.MapView;
 
+import java.io.File;
```
Comment (shellygill) [inline java/open-mobile-mappackage/src/main/java/com/esri/arcgisruntime/sample/openmobilemappackage/MainActivity.java]: As above, inconsistent with guidelines.

File: java/take-screenshot/src/main/java/com/esri/arcgisruntime/sample/takescreenshot/MainActivity.java [hunk 4]
```diff
@@ -16,33 +16,32 @@
...
 import android.widget.Toast;
-
 import com.esri.arcgisruntime.concurrent.ListenableFuture;
 import com.esri.arcgisruntime.mapping.ArcGISMap;
 import com.esri.arcgisruntime.mapping.Basemap;
 import com.esri.arcgisruntime.mapping.view.MapView;
 
+import java.io.File;
+import java.io.FileOutputStream;
+import java.io.IOException;
```
Comment (shellygill) [inline java/take-screenshot/src/main/java/com/esri/arcgisruntime/sample/takescreenshot/MainActivity.java]: As above.

**Comment by TADraeseke** [comment]
Thanks @shellygill - updated code style import layout. So that issue should stop appearing going forward.

**Pull request merged**